src/web/app/instructor-privilege.ts

```diff
@@ -1,12 +1,4 @@
-/**
- * Instructor privilege.
- */
-export interface InstructorPrivilege {
-  canModifyCourse: boolean;
-  canModifySession: boolean;
-  canModifyStudent: boolean;
-  canSubmitSessionInSections: boolean;
-}
+import { InstructorPrivilege } from '../types/api-output';
```

Comment:
Judging from the usage, you should move this file to `types` folder, and the constant below should be in `UPPER_CASE`, e.g. `DEFAULT_INSTRUCTOR_PRIVILEGE`.